config/helm/kustomization.yaml

```diff
@@ -7,6 +7,9 @@ images:
   - name: k8s.gcr.io/cluster-api-aws/cluster-api-aws-controller
     newName: "{{ .Values.infrastructure.image.domain }}/{{ .Values.infrastructure.image.name }}"
     newTag: "{{ .Values.tag }}"
+  - name: registry.k8s.io/cluster-api-aws/cluster-api-aws-controller
```

Comment:
do we need both? I thought they would all use the same registry domain

Reply:
I left it just in case they switch back in the next release. I've no idea what prompted the change. I don't see any harm in both.